sdcm/sct_events/setup.py

```diff
@@ -144,5 +144,12 @@ def enable_default_filters(sct_config: SCTConfiguration):  # pylint: disable=unu
     DbEventsFilter(db_event=DatabaseLogEvent.BACKTRACE, line='Rate-limit: suppressed').publish()
     DbEventsFilter(db_event=DatabaseLogEvent.WARNING, line='abort_requested_exception').publish()
 
+    # As per discussion in https://github.com/scylladb/scylla-enterprise/issues/4691#issuecomment-2348867638 and
+    # https://github.com/scylladb/scylla-cluster-tests/issues/8693#issuecomment-2358147285
+    # the 'aborting on shard' error is acceptable when RPC connections are dropped
+    EventsSeverityChangerFilter(new_severity=Severity.WARNING,
```

Comment:
instead of `EventsSeverityChangerFilter` maybe we could just add another event type that is verified before `aborting on shard` one?
@fruch WDYT?

Reply:
It's quite equivalent, in both cases it's running one more regex per line

The configuration like here is a bit more clear to my taste, adding a new type in the right place is a bit more tricky to do correctly.

Reply:
still, I think we could improve regex, as current one seems expensive